tests/runtime/test_env_vars.py

```diff
@@ -30,8 +29,7 @@ def test_env_vars_os_environ(temp_dir, box_class, run_as_openhands):
             obs.content.strip().split('\n\r')[0].strip() == 'BAZ'
         ), f'Output: [{obs.content}] for {box_class}'
 
-        runtime.close(rm_all_containers=False)
-        time.sleep(1)
+        _close_test_runtime(runtime)
```

Comment:
This doesn't call the boolean anymore, same below. It seems that now we set this to False in the app, not on tests.

Correct me if wrong, doesn't this mean we're not removing anymore obsolete containers from users' machines when they run into an error? If this is needed for tests, can we consider to keep it here, in tests, rather than in the app?